localstack-core/localstack/services/cloudformation/engine/v2/change_set_model.py

```diff
@@ -80,53 +113,123 @@ class ChangeSetTerminal(ChangeSetEntity, abc.ABC): ...
 
 
 class NodeTemplate(ChangeSetNode):
+    parameters: Final[NodeParameters]
+    conditions: Final[NodeConditions]
     resources: Final[NodeResources]
 
-    def __init__(self, change_type: ChangeType, resources: NodeResources):
-        super().__init__(change_type=change_type)
+    def __init__(
+        self,
+        scope: Scope,
+        change_type: ChangeType,
+        parameters: NodeParameters,
+        conditions: NodeConditions,
+        resources: NodeResources,
+    ):
+        super().__init__(scope=scope, change_type=change_type)
+        self.parameters = parameters
+        self.conditions = conditions
         self.resources = resources
 
 
+class NodeDivergence(ChangeSetNode):
```

Comment:
note for the future: it will be nice to get some docstrings indicating what these resources represent (possibly with an example from a template)